coderd/rbac/roles.go

```diff
@@ -369,6 +370,26 @@ type Permission struct {
 	Action       policy.Action `json:"action"`
 }
 
+func (perm Permission) Valid() error {
+	if perm.ResourceType == policy.WildcardSymbol {
+		// Wildcard is tricky to check. Just allow it.
```

Comment:
How would you even go about validating a wildcard? 🤔

Reply:
@johnstcn I could at least verify the action is in the list of all actions. Like `foo` is not the action :man_shrugging: